public/locales/en/translation.json

```
@@ -1,42 +1,41 @@
 {
   "pageTitle": {
     "ipfs": "IPFS Share",
-    "add": "Add Files",
+    "add": "Share Files",
     "download": "Download Files"
   },
   "header": "Share",
   "footer": "Licensed <1>CC-BY 3.0</1> except as <3>noted</3>",
   "info": {
     "add": {
-      "title": "Share files directly from your device",
+      "title": "Share files using IPFS",
+      "subtitle": "Share directly from your device — no cloud needed!",
       "labelAdd": "Add your files",
-      "copyAdd": "Anyone you give the link to will be able to get the files. Don't add sensitive or copyrighted files.",
-      "labelSend": "Send the share link",
-      "copySend": "IPFS gives the files a unique fingerprint called a CID that is used to find them again on the network.",
-      "labelKeepPage": "Keep this page open",
-      "labelKeepDaemon": "Keep your IPFS daemon running",
-      "copyKeep": "You are sharing the content from your device. Your friends can help you co-host the content by also keeping the share link open in their browser."
+      "copyAdd": "Add files to share them directly from your device over the IPFS network. Anyone with the share link can download your files, so don't add sensitive or copyrighted material.",
```

nit: I believe clarifying the CID role makes this note a bit closer to the technical truth: (anyone can listen on DHT or webrtc-star and log all uploaded CIDs):

```suggestion
      "copyAdd": "Add files to share them directly from your device over the IPFS network. Anyone with the unique CID identifier present in the share link can download your files, so don't add sensitive or copyrighted material.",
```

---

Good point - I don't want to be too technical for casual users but we can make this a little more specific. The "learn more" text can go into more detail about the CID concept once we have it opening in a modal. 

```suggestion
      "copyAdd": "Add files to share them directly from your device over the IPFS network. Anyone with the unique code in your share link can download your files, so don't add sensitive or copyrighted material.",
```